src/main/java/org/jabref/gui/groups/GroupDialog.java

```diff
@@ -345,6 +347,32 @@ public GroupDialog(DialogService dialogService, BasePanel basePanel, JabRefPrefe
             return null;
         });
 
+        nameField.textProperty().addListener(new ChangeListener<String>() {
```

Comment:
You can simplify the changeLIstener with lambdas: 
```suggestion
        nameField.textProperty().addListener((observable, oldValue, newValue) -> upddateComponent());
```

Reply:
Hi, Thanks for the suggestion. Working on it now. Will update soon

Reply:
@Siedlerchr The changes have been committed :)